capstone/src/main/resources/static/organizationsearch.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta>
+    <title>My Account</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+<body onload="searchOrg();">
+    <nav class="nav-bar">
+        <ul>
+            <li class="home"><a href="index.html">College Activity Finder</a></li>
+            <li><a href="">Events</a></li>
```

Comment:
Maybe we should switch index.html to under events and have the College activity finder act like a logo

Reply:
A lot of websites will use their logo as a way to get back to the home page, but I think either choice is fine

Reply:
What about putting an index link for both?